src/core/annotation.js

```diff
@@ -617,14 +623,63 @@ class MarkupAnnotation extends Annotation {
     super(parameters);
     const dict = parameters.dict;
 
-    if (!dict.has('C')) {
-      // Fall back to the default background color.
-      this.data.color = null;
+    this.data.isMarkup = true;
+
+    if (dict.has('IT')) {
```

Comment:
If I read the specification correctly, this shouldn't be necessary for the IRT/RT logic, so if that's the case please remove it from this patch to keep the functionality limited to IRT/RT. If need be, it can later be added in a follow-up patch.

Reply:
You are right. Removed.